provisioning/service/tests/Config/TpmAttestationTests.cs

```diff
@@ -69,26 +74,35 @@ public void TpmAttestation_SucceedOnNullStorageRootKey()
 
         [TestMethod]
         [TestCategory("DevService")]
-        public void TpmAttestation_ThrowsOnEnptyStorageRootKey()
+        public void TpmAttestation_SucceedOnEmptyStorageRootKey()
```

Comment:
Missing null test.

Reply:
`TpmAttestationTests` has a test for `TpmAttestation_SucceedOnNullStorageRootKey`